tests/stp_test.py

```diff
@@ -0,0 +1,703 @@
+        runner = CliRunner()
+        db = Db()
+        result = runner.invoke(
+            show.cli.commands["spanning-tree"].commands["statistics"].commands["vlan"], ["100"], obj=db)
+        print(result.exit_code)
+        print(result.output)
+        assert result.exit_code == 0
+        assert result.output == show_spanning_tree_statistics
+
+    def test_show_spanning_tree_bpdu_guard(self):
+        runner = CliRunner()
+        db = Db()
+        result = runner.invoke(show.cli.commands["spanning-tree"].commands["bpdu_guard"], [], obj=db)
+        print(result.exit_code)
+        print(result.output)
+        assert result.exit_code == 0
+        assert result.output == show_spanning_tree_bpdu_guard
+
+    def test_show_spanning_tree_root_guard(self):
+        runner = CliRunner()
+        db = Db()
+        result = runner.invoke(show.cli.commands["spanning-tree"].commands["root_guard"], [], obj=db)
+        print(result.exit_code)
+        print(result.output)
+        assert result.exit_code == 0
+        assert result.output == show_spanning_tree_root_guard
+
+    def test_disable_enable_global_pvst(self):
+        runner = CliRunner()
+        db = Db()
+
+        result = runner.invoke(config.config.commands["spanning-tree"].commands["disable"], ["pvst"], obj=db)
+        print("exit code {}".format(result.exit_code))
+        print("result code {}".format(result.output))
+        assert result.exit_code == 0
+
+        result = runner.invoke(config.config.commands["spanning-tree"].commands["enable"], ["pvst"], obj=db)
+        print("exit code {}".format(result.exit_code))
+        print("result code {}".format(result.output))
+        assert result.exit_code == 0
+
+        result = runner.invoke(config.config.commands["vlan"].commands["add"], ["100"], obj=db)
+        print(result.exit_code)
+        assert result.exit_code == 0
+
+        result = runner.invoke(
+            config.config.commands["vlan"].commands["member"].commands["add"], ["100", "Ethernet4"], obj=db)
+        print(result.exit_code)
+        assert result.exit_code == 0
+
+        result = runner.invoke(config.config.commands["spanning-tree"].commands["enable"], ["pvst"], obj=db)
+        print("exit code {}".format(result.exit_code))
+        print("result code {}".format(result.output))
+        assert result.exit_code != 0
+        assert "PVST is already configured" in result.output
+
+    def test_stp_validate_interface_params(self):
+        runner = CliRunner()
+        db = Db()
+
+        result = runner.invoke(config.config.commands["spanning-tree"].commands["disable"], ["pvst"], obj=db)
+        print("exit code {}".format(result.exit_code))
+        print("result code {}".format(result.output))
+        assert result.exit_code == 0
+
+        result = runner.invoke(
+            config.config.commands["spanning-tree"].commands["interface"].commands["enable"], ["Ethernet4"], obj=db)
+        print("exit code {}".format(result.exit_code))
```

Comment:
We should either all print(exit code {}.format...) or all print(result.exit_code) to avoid confusion for maintainers.

Reply:
Addressed